packages/pn-commons/src/utils/test.utility.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
```

Comment:
Forse non è necessario fare tutto ciò.
Si potrebbero testare propriamente tutti i componenti aggiunti in commons in modo tale che ricoprano le varie casistiche.
Per esempio si potrebbe testare:
- api ok -> mostra componente generico
- api errore -> mostra componente errore
-  ecc...

In questo modo non si ha bisogno nei componenti di personafisica o di pa di testare il funzionamento dei componenti di errore

Reply:
vale il detto nel commento susseguente (su SpecialContacts.test.tsx)